src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -248,11 +252,10 @@ def run_sequential_stages(
 
             # setup for next stage
             session = active_session()
-            session.reset_stage()
+            session.reset()
 
             # synchronize and clean up memory
             self.trainer.accelerator.wait_for_everyone()
-            self.trainer.model = get_session_model()
```

Comment:
was this code just redundant to start?

Reply:
No, I just find it to be clearer in the other location

Reply:
esp since we're trying to remove dependence on the trainer